projects/plugins/jetpack/modules/wordads/php/class-wordads-shortcode.php

```diff
@@ -32,34 +32,17 @@ public static function init() {
 	 * Our [wordads] shortcode.
 	 * Prints a WordAds Ad.
 	 *
-	 * @param array  $atts    Array of shortcode attributes.
-	 * @param string $content Post content.
-	 *
 	 * @return string HTML for WordAds shortcode.
 	 */
-	public static function handle_wordads_shortcode( $atts, $content = '' ) {
-		$atts = shortcode_atts( array(), $atts, 'wordads' );
-
-		return self::wordads_shortcode_html( $atts, $content );
-	}
-
-	/**
-	 * The shortcode output
-	 *
-	 * @param array  $atts    Array of shortcode attributes.
-	 * @param string $content Post content.
-	 *
-	 * @return string HTML output
-	 */
-	private static function wordads_shortcode_html( $atts, $content = '' ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
+	public static function handle_wordads_shortcode() {
 		global $wordads;
 
 		if ( empty( $wordads ) ) {
 			return '<div>' . __( 'The WordAds module is not active', 'jetpack' ) . '</div>';
 		}
 
-		$html = $wordads->insert_inline_ad( '' );
+		$html = '<div class="jetpack-wordad" itemscope itemtype="https://schema.org/WPAdBlock"></div>';
```

Comment:
_RE: Slack convo._ Just noting here that we would still want to bring [this change](https://github.com/Automattic/jetpack/pull/40733/files#diff-6a0a715a9c07ca5427d04aca1f28877c6185aba3d94f68988d5bc9f38ff771c7L541-L544) back 
i.e
```php
$content .= sprintf(
  '<div class="jetpack-wordad" itemscope itemtype="https://schema.org/WPAdBlock">%s</div>',
  $this->get_ad( 'inline', $ad_type )
);
```
_Considering that based on `itemscope itemtype="https://schema.org/WPAdBlock` it only makes sense to have this wrap the actual ad as the schema is meant to help search engines or crawlers to identify the actual ad section._